UDx-container/vsdk-vertica

```diff
@@ -0,0 +1,165 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Unlike the other vsdk-* commands, this command actually has
+# command-line arguments.
+
+# However, to be consistent with the other vsdk-* commands, it
+# understands the same environment variables they do.
+
+# For example, $OSTAG-$VERTICA_VERSION is used to define the container
+# tag (e.g., verticasdk:centos-11.0.0-0).  So, if you have multiple
+# verticavsdk containers lying around, you can select among them by
+# defining these variables in the environment.
+#
+# OSTAG: one of { centos, ubuntu } --- selects the container OS
+#       version (presumably different from the host operating system,
+#       as otherwise the script just picks it up from
+#       /etc/os-release).
+# VERTICA_VERSION: something like 11.0.0-0
+#
+# In addition, there is these environment variables:
+#
+# VSDK_MOUNT: an optional space-separated list of host directories to
+#       mount in the UDx container --- that is, "I'm going to want to
+#       refer to files in these directories in, for example, my
+#       makefile, or as include directories in code to be compiled"
+#
+# VSDK_ENV: names a file filled with variable definitions.  This uses
+#       the --env-file option of the "docker run" command, so the file
+#       needs to be formatted for that option
+
+# The user can override the OS selection by defining OSTAG in the
+# environment (mostly for testing)
+. /etc/os-release
+
+case $ID in
+    rhel) OSTAG=${OSTAG:=centos} ;;
+    ubuntu) OSTAG=${OSTAG:=ubuntu} ;;
+    *) OSTAG=${OSTAG:=centos} ;;
+esac
+case $OSTAG in
+    centos)
+        BINDIR=/usr/bin 
+        ALSO_IN_PATH=/bin
+        ;;
+    ubuntu)
+        BINDIR=/bin
+        ALSO_IN_PATH=/usr/bin
+        ;;
+esac
+
+V_VERSION=${VERTICA_VERSION:=11.0.0-0}
+IMAGE_NAME=verticasdk
+TAG=$OSTAG-$V_VERSION
+
+CONTAINER_NAME=verticasdk
+DVOL=verticasdk-data 
+
+VSDK_VOLUMES=""
+if [ "$VSDK_MOUNT"x != x ]; then
+    for dir in $VSDK_MOUNT; do
+        VSDK_VOLUMES="$VSDK_VOLUMES -v $dir:$dir"
+    done
+fi
+
+VSDK_ENV_FILE_OPTION=""
+if [ "$VSDK_ENV"x != x ]; then
+    VSDK_ENV_FILE_OPTION="--env-file $VSDK_ENV"
+fi
+
+CONTAINER_PATH=/opt/rh/rh-nodejs8/root/usr/bin
+CONTAINER_PATH=$CONTAINER_PATH:/opt/rh/llvm-toolset-7.0/root/usr/bin
+CONTAINER_PATH=$CONTAINER_PATH:/opt/rh/llvm-toolset-7.0/root/usr/sbin
+CONTAINER_PATH=$CONTAINER_PATH:/opt/rh/rh-git218/root/usr/bin
+CONTAINER_PATH=$CONTAINER_PATH:/opt/rh/devtoolset-7/root/usr/bin
+CONTAINER_PATH=$CONTAINER_PATH:/usr/lib64/ccache
+CONTAINER_PATH=$CONTAINER_PATH:${BINDIR}
+CONTAINER_PATH=$CONTAINER_PATH:${ALSO_IN_PATH}
+CONTAINER_PATH=$CONTAINER_PATH:/sbin
+CONTAINER_PATH=$CONTAINER_PATH:/usr/sbin
+CONTAINER_PATH=$CONTAINER_PATH:/usr/local/bin
+CONTAINER_PATH=$CONTAINER_PATH:/usr/local/sbin
+
+user_id=`id -u`
+HOSTNAME=vsdk-$USER
```

Comment:
Do we use this variable?  Is the intent to set the hostname in the container.

Reply:
I should check to see if there is a lint for shell scripts.